src/main/java/br/com/harbitech/school/category/CategoryForm.java

```diff
@@ -0,0 +1,93 @@
+package br.com.harbitech.school.category;
+
+import br.com.harbitech.school.subcategory.Subcategory;
+
+import javax.validation.constraints.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CategoryForm {
+    private Long id;
+    @NotBlank(message = "Por favor insira o nome da categoria")
+    @Size(max = 70, message = "Ops! O nome de uma categoria não pode ter mais do que 70 caracteres")
```

Comment:
```suggestion
    @NotBlank(message = "{category.name.required}")
    @Size(max = 70, message = "{category.name.size.max}")
```

E colocar as mensagens em um arquivo `src/main/resources/ValidationMessages.properties`:

```properties
category.name.required = Por favor insira o nome da categoria
category.name.size.max = Ops! O nome de uma categoria não pode ter mais do que {max}  caracteres

```

Reply:
Muitooo obrigada! ficou bem melhor